mobile/src/components/uam-configure-group/components/permissions-table/permissions-table.tsx

```diff
@@ -62,6 +62,7 @@ const PermissionsTable: FC<Props> = ({
         columns={permissionColumns}
         data={permissionRows}
         columnWidthArr={[50, 200, 100]}
+        showScrollbar={false}
```

Comment:
Why did you hide it there?

Reply:
I've wrote in PR description. We don't have scroll on this table, and it looks like long dark line on the bottom of the table